decred/decred/dcr/txscript.py

```diff
@@ -1836,6 +1844,10 @@ def payToStakeSHScript(addr, stakeCode):
     Returns:
         ByteArray: The script that pays to the script hash of the address.
     """
+    if not isinstance(addr, addrlib.AddressScriptHash):
+        raise DecredError(f"address is not an AddressScriptHash")
+    if stakeCode not in (opcode.OP_SSTX, opcode.OP_SSTXCHANGE):
+        raise DecredError(f"stake code is not sstx or sstxchange: {hex(stakeCode)}")
```

Comment:
Same here. Should allow `OP_SSRTX` and use this in `payToStakeSHScript` in `payToSSRtxSHDirect`.